--- a/src/gitversion/gitversion.cmake
+++ b/src/gitversion/gitversion.cmake
@@ -1,7 +1,7 @@
 set(DIR_OF_GITVERSION_TOOL "${CMAKE_CURRENT_LIST_DIR}" CACHE INTERNAL "DIR_OF_GITVERSION_TOOL")
 
 function (get_git_version OUTPUT_VARIABLE)
-    EXECUTE_PROCESS(COMMAND python ${DIR_OF_GITVERSION_TOOL}/getversion.py
+    EXECUTE_PROCESS(COMMAND python3 ${DIR_OF_GITVERSION_TOOL}/getversion.py
         WORKING_DIRECTORY ${DIR_OF_GITVERSION_TOOL}
         OUTPUT_VARIABLE VERSION
         ERROR_VARIABLE error
